### ISSUE TITLE
The rat to exe 

### ISSUE THREAD

**Issue by KappaAppak** [pythondemon/Discord-Rat#2]
I can transform the rat to exe, please ?

**Comment by pythondemon** [comment]
pip install auto-py-to-exe
auto-py-to-exe
or watch a video in youtube

**Comment by pythondemon** [comment]
@KappaAppak

**Comment by KappaAppak** [comment]
yes, but when I turn it into an exe, I get an error that I can't run it

**Comment by KappaAppak** [comment]
yes, but when I turn it into an exe, I get an error that I can't run it

**Comment by d3-d7** [comment]
> yes, but when I turn it into an exe, I get an error that I can't run it 

@KappaAppak 

Send image

**Comment by KappaAppak** [comment]
![image](https://user-images.githubusercontent.com/71730407/210260035-50d4d100-fc2d-46c0-86b7-eb6af89184d5.png)

**Comment by KappaAppak** [comment]
?

**Comment by kalry-cmd** [comment]
guys who can help me with install this rat
